api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +178,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+	
+	@Test
+	public void valid_shouldRejectNumericAllergenValue() {
```

Comment:
Remove the unwanted spaces in this method

Reply:
@HerbertYiga ......indeed i also wanted to remove them but......according to the styling of the other Tests...........I just wanted to follow the Order.............Is it worth it.......to remove them or......just leave as the other tests

Reply:
It is true , some other tests have spaces, but only were appropriate .Not everywhere

Reply:
Thanks, @tendomart I think all issues a now solved........